docs/source/yelpsoa_configs.rst

```diff
@@ -1069,6 +1069,33 @@ internally at Yelp.
      delay. Note that due to Envoy's current definition of *percentage* as an
      integer, this cannot be specified as a floating-point number.
 
+Error Alerting
+``````````````
+
+These keys provide optional overrides for the default alerting behaviour.
+
+ * ``monitoring``: Override default alerting behaviour. For example: ::
+
+      main:
+        monitoring:
+          team: frontend
+          slack_channel: "notifications"
+          jira_project: "FRONTEND"
+          ticket: false
+          page: false
+          page_nonprod: true
+          error_threshold_ration: 0.02
+          minimum_error_rps: 10
+    - ``team``: Override the default team for alerting.
+    - ``slack_channel``: Override the default slack channel for alerting.
```

Comment:
monitoring.yaml allows for multiple slack channels - if we're using monitoring.yaml as a base then we'll probably need to deal with that :p

Reply:
Updated the text, we will take the first channel from that list as we can only alert a singe channel

Reply:
@nemacysts dod you mean deploy.yaml? Theres no slack_channels list in the monitoring_schema

Reply:
ok comparing the docs with the monitoring schema there are a load of available fields that arent in the schema file so I guess validation is not happening currently for monitoring.yaml?  We are going to proceed taking the first channel in the array as it is currently specified in the docs